### PR TITLE
Feature/rule docs

### PR DESCRIPTION
**Pull request by tyler-mairose-sp**

### CODE REVIEW

File: products/idn/docs/identity-now/rules/cloud-rules/index.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+id: cloud-executed-rules
+title: Cloud Executed Rules
+pagination_label: Cloud Executed Rules
+sidebar_label: Cloud Executed Rules
+sidebar_position: 1
+sidebar_class_name: cloudExecutedRules
+keywords: ["cloud", "rules"]
+description: Overview of cloud executed rules
+slug: /docs/rules/cloud-rules
```
Comment (jordan-violet-sp) [inline products/idn/docs/identity-now/rules/cloud-rules/index.md:10]: This slug should just be `/docs/rules`

File: products/idn/docs/identity-now/rules/index.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
+---
+id: rules
+title: Rules
+pagination_label: Rules
+sidebar_label: Rules
+sidebar_position: 2
+sidebar_class_name: rules
+keywords: ["rules"]
+description: Rule guide for developing rules in Identity Now.
```
Comment (jordan-violet-sp) [inline products/idn/docs/identity-now/rules/index.md:9]: I wouldn't use the word _guide_ here, as we use Guides to define guides/tutorials/walkthroughs on how to do something. I'd say _Documentation_ instead.

File: products/idn/docs/identity-now/rules/index.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
+---
+id: rules
+title: Rules
+pagination_label: Rules
+sidebar_label: Rules
+sidebar_position: 2
```
Comment (jordan-violet-sp) [inline products/idn/docs/identity-now/rules/index.md:6]: sidebar_position should be 3

**Review by jordan-violet-sp — COMMENTED** [review]
Made some comments. Also add James as a reviewer to review the docs grammar/formatting and overall content.

File: products/idn/docs/identity-now/rules/index.md (hunk 2 above)
Comment (tyler-mairose-sp) [reply products/idn/docs/identity-now/rules/index.md:9]: Updated.

**Review by james-haytko-sp — APPROVED** [review]

**Pull request merged**